internal/aez/ccd_data.go

```diff
@@ -0,0 +1,169 @@
+package aez
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/innosat-mats/rac-extract-payload/internal/ccsds"
+)
+
+// WDWMode describes the CCD WDW parameter
+type WDWMode uint8
+
+// WDWModeManual = 0, check WDWOV for overflows
+//
+// WDWModeAutomatic, window selected depending on input data
+const (
+	WDWModeManual    WDWMode = 0
+	WDWModeAutomatic WDWMode = 1
+)
+

```

Comment:
If you do like this VSC will help you with the constants (does not work with block comments):
```suggestion
const (
	// WDWModeManual = 0, check WDWOV for overflows
	WDWModeManual    WDWMode = 0
	// WDWModeAutomatic, window selected depending on input data
	WDWModeAutomatic WDWMode = 1
)

```

Reply:
mine seemed to work too, but nicer to just get info on the one being used rather than all of them I suppose.